code/game/objects/items/spear.dm

```diff
@@ -130,7 +130,15 @@
 	. = ..()
 	if(!proximity)
 		return
-	if(wielded)
+	if(wielded && istype(AM))
+		if(AM.resistance_flags & INDESTRUCTIBLE) //due to the lich incident of 2021, embedding grenades inside of indestructible structures is forbidden
+			return
+		if(ismob(AM))
+			var/mob/mob_target = AM
+			if(mob_target.status_flags & GODMODE) //no embedding grenade phylacteries inside of ghost poly either
+				return
+		if(iseffect(AM)) //no accidentally wasting your moment of glory on graffiti
+			return
 		user.say("[war_cry]", forced="spear warcry")
 		explosive.forceMove(AM)
```

Comment:
This is the real problem. The grenade shouldn't be moved inside the target atom, but on its turf.

Reply:
you're driving it into them using your spear

Reply:
Then have the grenade properly embedded into mobs instead of forcemoving it. As for movables, just move the grenade on their tile for the time being.

Reply:
but that screws over advanced release grenade elances

like, significantly enough to affect balance

my gbp can't take that, man